src/lib.rs

```diff
@@ -48,7 +48,7 @@ extern crate bincode;
 extern crate config_file_handler;
 #[macro_use]
 extern crate lazy_static;
-#[macro_use]
+#[cfg_attr(test, macro_use)]
```

Comment:
ready to go @ustulation, although safe_vault will break if we release a new version without merging https://github.com/maidsafe/safe_vault/pull/606 (which depends on routing master, or a new version)